primitives/npos-elections/src/lib.rs

```diff
@@ -133,6 +133,10 @@ pub enum Error {
 	ArithmeticError(&'static str),
 	/// The data provided to create support map was invalid.
 	InvalidSupportEdge,
+	// TODO this should be removed once PhragMMS no longer uses &str errors and is migrated to
```

Comment:
I think this should be fixed in this PR. The clear reason for me is that if fix this now, then we're actually _removing code form this PR_ (since it makes things simpler), and thus making it to review and maintain.

Reply:
to elaborate more, if to deliver feature X you can _omit_ typing 50 extra lines in this PR, sure, that's worth considering. But it feels like the opposite here, we're adding code that's going to be removed later.